app/test_main.py

```diff
@@ -1 +1,43 @@
-# write your code here
+from unittest import mock
+import pytest
+from app.main import can_access_google_page
+
+
+@pytest.fixture()
+def mocked_valid_url():
+    with mock.patch("app.main.valid_google_url") as mock_valid_url:
+        yield mock_valid_url
+
+
+@pytest.fixture()
+def mocked_has_internet_connection():
+    with mock.patch("app.main.has_internet_connection") as mock_has_internet:
+        yield mock_has_internet
+
+
+def test_all_works(mocked_valid_url, mocked_has_internet_connection):
+    mocked_valid_url.return_value = True
+    mocked_has_internet_connection.return_value = True
+    assert can_access_google_page("www.google.com/") == "Accessible", \
```

Comment:
Sorry for bringing it up again, but now I have an actual quote from our mentor Oleksii to reference:
> Avoid using `\`. Use `( )`

Reply:
thx for your review, I've fixed it recently :))